massa-consensus/src/ledger.rs

```diff
@@ -77,6 +77,7 @@ impl LedgerChanges {
     }
 
     /// clone subset
+    #[must_use]
```

Comment:
What does that do ?

Reply:
It helps us to indicate that a function return a value that must be use! https://rust-lang.github.io/rfcs/1940-must-use-functions.html (meaning that if a developer forget to assign that return value, it's certainly a mistake)